src/main/java/demo/ClusterCalphas.java

```diff
@@ -0,0 +1,52 @@
+package demo;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.broadcast.Broadcast;
+import org.apache.spark.mllib.clustering.PowerIterationClustering;
+import org.apache.spark.mllib.clustering.PowerIterationClusteringModel;
+import org.rcsb.mmtf.spark.SparkUtils;
+import org.rcsb.mmtf.spark.data.Segment;
+import org.rcsb.mmtf.spark.data.StructureDataRDD;
+
+import scala.Tuple2;
+import scala.Tuple3;
+
+/**
+ * Demo code of clustering C-alpha chains.
+ * @author Anthony Bradley
+ *
+ */
+public class ClusterCalphas {
+
+	/**
+	 * 
+	 * @param args
+	 * @throws IOException
+	 */
+	public static void main(String[] args) throws IOException {
+		// Get the underlying C-alpha chains
+//		List<Tuple2<String, Segment>> calphaChains = SparkUtils.getCalphaChains(new String[] {"4CUP","4CUQ","4CUT","4CUR","4CUS","1STP"}).filterMinLength(10).getSegmentRDD().collect();
+		List<Tuple2<String, Segment>> listChains = new StructureDataRDD("/Users/anthony/full").sample(0.01).getCalpha().getSegmentRDD().collect();
+		int numChains = listChains.size();
+		Broadcast<List<Tuple2<String, Segment>>> calphaChains = SparkUtils.getSparkContext().broadcast(listChains);
```

Comment:
Again, feels wrong to have to reduce two lines earlier, just so you can broadcast this here.